spec/test_app_templates/lib/generators/test_app_generator.rb

```diff
@@ -0,0 +1,14 @@
+        require 'rails/generators'
```

Comment:
Why the indentation for this file?

Reply:
EngineCart made me do it. It gets generated when you run `engine_cart:prepare`.